source/simulator/checkpoint_restart.cc

```diff
@@ -263,6 +263,8 @@ namespace aspect
   void Simulator<dim>::create_snapshot()
   {
     TimerOutput::Scope timer (computing_timer, "Create snapshot");
+    walltime_from_resume += walltime.wall_time();
+    walltime.reset();
```

Comment:
```suggestion
    walltime.restart();
```

Reply:
Thanks. The first time outputted now matches that from deal. But on a restart the second wallclock time is 200s later than the deal times added. I will try with these "walltime_from_resume += walltime.wall_time(); walltime.restart();" should go straight after the new print statements.

Reply:
that didn't work. I'm not too sure where "walltime_from_resume += walltime.wall_time(); walltime.restart();" should be moved to. Any ideas please?

Reply:
This seems to be the right place to me. What happens?

Reply:
The first time outputted matches that from deal. But on a restart the second wallclock time is 200s later than the first and second deal.II times added.